react/client/src/styles/utilityStyles.ts

```diff
@@ -0,0 +1,29 @@
+import { makeStyles, createStyles } from '@material-ui/core';
+
+const useUtilityStyles = makeStyles((theme) =>
+  createStyles({
+    // I think I named this, but I'd like to rename this better because it's not simply a flexbox container, if anyone has ideas - Alex
+    flex: {
```

Comment:
Looking at it, we're just using it for buttons to be in the size and location -- let's call it `buttonContainer`